packages/lab/src/components/Flow/stories/Usage.stories.mdx

```diff
@@ -0,0 +1,427 @@
+It leverages the [ReactFlow](https://reactflow.dev/) library under the hood.
+
+Summary:
+
+- [Creating flows](#creating-flows)
+  - [The Sidebar](#the-sidebar)
+  - [Layout](#layout)
+    - [Background](#background)
+    - [Controls](#controls)
+    - [Minimap](#minimap)
+  - [Custom nodes and node types](#custom-nodes-and-node-types)
+  - [Inputs and Outputs](#inputs-and-outputs)
+  - [Parameters](#parameters)
+  - [Actions](#actions)
+    - [Default Actions](#default-actions)
+    - [Custom Actions](#custom-actions)
+  - [Custom Content](#custom-content)
+- [Custom Node Example](#custom-node-example)
+
+# Creating flows <a id="creating-flows" />
+
+To use the `HvFlow` component you'll need to define:
+
+- Node groups
+- Node types
+
+Optionally you can also define:
+
+- Initial nodes
+- Initial edges
+- Default actions that all nodes will have
+
+## The Sidebar <a id="the-sidebar" />
+
+To create the groups of nodes in the sidebar you'll have to create node groups. These should have a set of basic properties:
+
+```typescript
+interface HvFlowNodeGroup {
+  label: string;
+  description?: string;
+  color?: HvColorAny;
+  icon?: React.ReactNode;
+}
+```
+
+Example:
+
+```typescript
+// Node groups
+type NodeGroups = "assets" | "models";
+
+const nodeGroups = {
+  assets: {
+    label: "Assets",
+    color: "cat3_80",
+    description: "This is my description for assets.",
+    icon: <Heart />,
+  },
+  models: {
+    label: "ML Model",
+    color: "cat1_80",
+    description: "This is my description for models.",
+    icon: <Favorite />,
+  },
+} satisfies HvFlowProps<any, NodeType, NodeGroups>["nodeGroups"];
+```
+
+You can then create the sidebar by using the `HvFlowSidebar` component as a prop of the `HvFlow` component:
+
+```tsx
+<HvFlow
+  nodeGroups={nodeGroups}
+  (...)
+  sidebar={
+    <HvFlowSidebar
+      title="Add Node"
+      description="Please choose within the options below"
+      open={open}
+      onClose={() => setOpen(false)}
+    />
+  }
+
+>
+  (...)
+</HvFlow>
+```
+
+## Layout <a id="layout" />
+
+There a few controls that you can add to your flow to control its layout and some functionality. All of these are components that you
+should pass as children of the `HvFlow` component.
+
+### Background <a id="background" />
+
+To add a background to the flow you should use the `<HvFlowBackground />` component. This is a wrapper around the
+[Background](https://reactflow.dev/docs/api/plugin-components/background/) component of `reactflow` and has the same API.
+
+### Controls <a id="controls" />
+
+To add the controls panel to the flow you should use the `<HvFlowControls />` component. This, again, uses the
+[Controls](https://reactflow.dev/docs/api/plugin-components/controls/) component from `reactflow` but you should check [our own
+documentation](?path=/docs/lab-flow--docs) for the `HvFlowControls` component also.
```

Comment:
This link is not working. I think it should be `./?path=/docs/lab-flow--docs` if I'm not mistaken.